### PR TITLE
Add builder course 1

### PR DESCRIPTION
**Pull request by nygeog**
@ohasselblad @ztephm 

Here is a version of the builder course Vizualization 1: UI 

I think the main concerns I have are style and consistency and whether the images make sense in the context of the language and also in reference to Dave's videos. 

Please comment as much as you can as this review will help style the following vid's

### CODE REVIEW

**Comment by ztephm** [comment]
Good job! 
* I think your explanations were clear and a good length, not too wordy.
* I liked your style of highlighting here!

  9-sql-pane

I think we should keep the style consistent across all feature-highlight images though - so for example use the same technique/colors for highlighting in the Layer Options section "customer" image. If you meant "red" to indicate an action instead, like click, then I suggest replacing that with an image with a cursor over what you are clicking...or even an animated gif showing the action.

  We should also keep highlighting consistent across lessons. Right now we're using this style in other lessons:

  highlight

  - @ohasselblad and @carlostallon do you have opinions about which style of feature highlighting to use? 

* You brought up an interesting situation: "Data and Maps views" are mentioned in the first paragraph to mean the main Datasets page and main Maps page visible when you first log into your account? If so I would like to change "view" to "page" just because Data View and Map View are specific UI names in the Editor for once you're inside a Map or Dataset. Let me know if this is what we're officially calling these pages now in the Builder though! - since this UI is going away:

  screen shot 2016-07-22 at 7 25 00 pm

File: _app/_builder-intro/01-builder-visualization-1-ui.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Builder Introduction — Visualization Part 1: UI"
+permalink: /courses/builder-intro/builder-viz-p1-ui/
+tweet_text: "Have 5 min free to learn the new CARTO Builder UI? Check out Visualization Part 1: UI. @cartoHQ"
```
Comment (ohasselblad) [inline _app/_builder-intro/01-builder-visualization-1-ui.md]: now `@CARTO` twitter

**Comment by ztephm** [comment]
hi @ohasselblad @nygeog i added widgets writeup, but without images for now: I commented out where they need to be put later tho.
